README.md

```diff
@@ -45,6 +45,8 @@ Firefox WebExtension Experiments APIs providing capabilities for study add-ons t
 │   └── shield-study-helper-addon/
 ├── package-lock.json
 ├── package.json
+├── test/
```

Comment:
The move of test and test-addon to the root directory implies that small-study and other future examples all will be tested through the files in these directories. If that will be the case, fine. If not, having `test` which only tests `test-addon` is confusing. Add tests for `small-study` and see how it feels?

Reply:
Added https://github.com/mozilla/shield-studies-addon-utils/issues/172

Reply:
I claim the 'test-addon' is for testing these things:
1.  The public api
2.  Prove that we can do selenium / webdriver tests.

You have implemented both.  I think the `test-addon` is less about demonstrating the api.

I claim the 'test' directory is to actually test the API, and the mechanism for doing so is currently via addon using webDriver.  

I think it's less confusing (after feedback) to move the test-addon out of examples.

Reply:
We can revisit this after #172 is done. Maybe it is not confusing to have two different styles of test directories.